providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/triggers/job.py

```diff
@@ -55,7 +55,7 @@ def __init__(
         self,
         job_name: str,
         job_namespace: str,
-        pod_name: str,
```

Comment:
Seems like a breaking change - not entirely sure that it is user facing, but it could potentially break customizations.
Let's deprecate it slowly?

Reply:
Ok, sounds reasonable! Thank you for your comment, I´ll update my PR

Reply:
Done!